src/callbacks_step/glm_speed.jl

```diff
@@ -6,26 +6,30 @@
 #! format: noindent
 
 """
-    GlmSpeedCallback(; glm_scale=0.5, cfl)
+    GlmSpeedCallback(; glm_scale=0.5, cfl, semi_indices=())
 
 Update the divergence cleaning wave speed `c_h` according to the time step
 computed in [`StepsizeCallback`](@ref) for the ideal GLM-MHD equations.
 The `cfl` number should be set to the same value as for the time step size calculation. The
 `glm_scale` ensures that the GLM wave speed is lower than the fastest physical waves in the MHD
 solution and should thus be set to a value within the interval [0,1]. Note that `glm_scale = 0`
 deactivates the divergence cleaning.
+In case of coupling semidiscretizations, specify for which semi_index the divergence
```

Comment:
```suggestion
In case of coupled semidiscretizations, specify for which semi_index the divergence
```
Could you also clarify what is meant by `semi_index`? From context I think it is the local index of a particular semidiscretization but best to write an additional sentence and remove ambiguity.

Reply:
Added some clarification about semi_index.